akka-actor-tests/src/test/scala/akka/actor/LocalActorRefProviderSpec.scala

```diff
@@ -112,11 +112,13 @@ class LocalActorRefProviderSpec extends AkkaSpec(LocalActorRefProviderSpec.confi
       expectTerminated(a)
       // the fields are cleared after the Terminated message has been sent,
       // so we need to check for a reasonable time after we receive it
-      awaitAssert({
-        val childProps2 = child.asInstanceOf[LocalActorRef].underlying.props
-        childProps2 should not be theSameInstanceAs(childProps1)
-        (childProps2 should be).theSameInstanceAs(ActorCell.terminatedProps)
-      }, 1 second)
+      awaitAssert(
+        {
```

Comment:
`({` seems more compact.

Reply:
Same as the other remark I guess. 
(<long block>, 1 second)
I understand why it gets split over lines and tend to agree with the splitting.